suite-native/module-onboarding/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './screens/WelcomeScreen';
```

Comment:
do you need export this?

Reply:
[1d456aa](https://github.com/trezor/trezor-suite/pull/7889/commits/1d456aae554fa46269c007c6047352dfb9d28407)